examples/ensemble/plot_isolation_forest.py

```diff
@@ -1,67 +1,114 @@
 """
-==========================================
+=======================
 IsolationForest example
-==========================================
+=======================
 
 An example using :class:`~sklearn.ensemble.IsolationForest` for anomaly
 detection.
 
-The IsolationForest 'isolates' observations by randomly selecting a feature
-and then randomly selecting a split value between the maximum and minimum
-values of the selected feature.
+The IsolationForest 'isolates' observations by randomly selecting a feature and
+then randomly selecting a split value between the maximum and minimum values of
+the selected feature.
 
-Since recursive partitioning can be represented by a tree structure, the
-number of splittings required to isolate a sample is equivalent to the path
-length from the root node to the terminating node.
+Since recursive partitioning can be represented by a tree structure, the number
+of splittings required to isolate a sample is equivalent to the path length from
+the root node to the terminating node.
 
-This path length, averaged over a forest of such random trees, is a measure
-of normality and our decision function.
+This path length, averaged over a forest of such random trees, is a measure of
+normality and our decision function.
 
-Random partitioning produces noticeable shorter paths for anomalies.
-Hence, when a forest of random trees collectively produce shorter path lengths
-for particular samples, they are highly likely to be anomalies.
+Random partitioning produces noticeable shorter paths for anomalies. Hence, when
+a forest of random trees collectively produce shorter path lengths for
+particular samples, they are highly likely to be anomalies.
 
 """
 
+# %%
+# Data generation
+# ---------------
+#
+# We generate two clusters (each one containing `n_samples`) by randomly
+# sampling the standard normal distribution as returned by `numpy.random.randn`.
```

Comment:
This should work since we link to numpy as well.

```suggestion
# sampling the standard normal distribution as returned by :func:`numpy.random.randn`.
```